modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -42,10 +43,20 @@ public static String[] getExpectedMessages() throws Exception {
 
 			String from = jsonObject.getString("from");
 
-			if (from.contains(StringPool.OPEN_PARENTHESIS) &&
-				!jsonObject.getBoolean("skipParametersValidation")) {
+			String[] validExtensions = JSONUtil.toStringArray(
+				jsonObject.getJSONArray("validExtensions"));
+
+			if (validExtensions.length==0) {
```

Comment:
@MicaelleSilvaa, please run source formatter to adjust this code according Liferay Code Convention. If you already did it in another commit from this pull, please merge this 2 commits. :wink: